### PR TITLE
[RPS-19] Deploy Hippo Delivery stack

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
To start/update Hippo Delivery stack simply run
```
make stack ROLE=hippo_delivery
```

### CODE REVIEW

File: ansible/roles/service/hippo_delivery/files/hippo_delivery.yml [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    Type: Number
+    Default:  1
+
+  InstanceMaxSize:
+    Description: Maximum number of EC2 instances to launch in Asg
+    Type: Number
+    Default: 6
+
+  InstanceMonitoring:
+    Description: Should the detailed instance monitoring be enabled
```
Comment (ivorc) [inline ansible/roles/service/hippo_delivery/files/hippo_delivery.yml:26]: Don't we want this on?

**Review by ivorc — COMMENTED** [review]
Just one question

File: ansible/roles/service/hippo_delivery/files/hippo_delivery.yml (hunk 1 above)
Comment (wojtek-oledzki) [reply ansible/roles/service/hippo_delivery/files/hippo_delivery.yml:26]: it's extra $$ on dev and stg. We might not even need this on production.

Comment (ivorc) [reply ansible/roles/service/hippo_delivery/files/hippo_delivery.yml:26]: Got you, fine.

**Pull request merged**